angular2/src/angular2TextMask.ts

```diff
@@ -74,5 +74,5 @@ export class MaskedInputDirective implements OnInit, ControlValueAccessor{
   exports: [MaskedInputDirective],
   imports: [CommonModule]
 })
-export class TextMaskModule {
-}
+export class TextMaskModule {}
+export {conformToMask} from 'text-mask-core/dist/textMaskCore'
```

Comment:
You could use it directly in the templates. Otherwise you need to import the function in your class then expose it in a class variable to use it in the template.

With the pipe your just do `{{ myValue | conformToMask }}` and you are done.